<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoRenderViewInfo</class>
 <widget class="QWidget" name="VideoRenderViewInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>31</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="name_label">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>212</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="net_btn">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="mic_btn">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
